applications/SolidMechanicsApplication/python_scripts/restart_process.py

```diff
@@ -39,7 +39,7 @@ def __init__(self, Model, custom_settings ):
         self.model = Model
 
         # set up output frequency and format
-        self.output_frequency  = self.settings["output_frequency"].GetDouble()
+        self.output_frequency  = self.settings["output_interval"].GetDouble()
```

Comment:
this process is heavily used, please also make it backward compatible

I am surprised this even works in the tests, I thought I had tested it well

Reply:
I have applied the suggested changes